en/docs/guides/Provisioning/outbound-provisioning.md

```diff
@@ -0,0 +1,10 @@
+# Outbound Provisioning
+
+This topic provides instructions on how to provision users to a trusted identity provider from the WSO2 Identity Server. A trusted identity provider is basically an identity provider that supports inbound provisioning. It can be Google, Salesforce, another Identity Server, etc. For more information, see [Outbound provisioning](insert-concepts-link).
```

Comment:
A trusted identity provider is basically an identity provider that supports inbound provisioning such as Google, Salesforce...